browser/brave_browser_main_extra_parts.cc

```diff
@@ -14,4 +15,7 @@ BraveBrowserMainExtraParts::~BraveBrowserMainExtraParts() {
 
 void BraveBrowserMainExtraParts::PreMainMessageLoopRun() {
   brave::AutoImportMuon();
+#if BUILDFLAG(BUNDLE_WIDEVINE_CDM)
+  RegisterWidevineCdmToCdmRegistry();
```

Comment:
rename -> `MaybeRegister...` ?

Reply:
And also skip `ToCdmRegistry`, sounds obvious :)

Reply:
Thanks. it's more clear 👍